### PR TITLE
Update sFlow HLD

### PR DESCRIPTION
**Pull request by GarrickHe**
* Update sFlow HLD to reflect migration to OpenConfig YANG.
* Update sFlow CLI.
* Update sFlow error handling.

Signed-off-by: Garrick He <garrick_he@dell.com>

### CODE REVIEW

File: doc/mgmt/SONiC_Management_Framework_sFlow.md [hunk 1]
```diff
@@ -161,26 +170,26 @@ Syntax:
 
 port# : [0 - 65535] (default is: 6343)
 
-`sflow collector <collector name> <IPv4/IPv6 address> [port #]`
+`sflow collector <IPv4/IPv6 address> [port #]`
 
 ```
-sonic(config)# sflow collector col1 1.1.1.1
+sonic(config)# sflow 1.1.1.1
```
Comment (michaelli10) [inline doc/mgmt/SONiC_Management_Framework_sFlow.md]: Still need the "collector" keyword here.

File: doc/mgmt/SONiC_Management_Framework_sFlow.md [hunk 2]
```diff
@@ -161,26 +170,26 @@ Syntax:
...
 sonic(config)#
 ```
 
 ##### Add sFlow Collector with port number
 ```
-sonic(config)# sflow collector col2 1.1.1.2 4451
+sonic(config)# sflow 1.1.1.2 4451
```
Comment (michaelli10) [inline doc/mgmt/SONiC_Management_Framework_sFlow.md]: Same here.  Still need the "collector" keyword.

File: doc/mgmt/SONiC_Management_Framework_sFlow.md (hunk 1 above)
Comment (GarrickHe) [reply doc/mgmt/SONiC_Management_Framework_sFlow.md]: Will fix. Thank you.

File: doc/mgmt/SONiC_Management_Framework_sFlow.md (hunk 2 above)
Comment (GarrickHe) [reply doc/mgmt/SONiC_Management_Framework_sFlow.md]: Will fix. Thank you.

**Pull request merged**